desktop/analytics/account_creation.js

```diff
@@ -69,6 +69,18 @@ if (Cookies.get('analytics-signup')) {
   }
 }
 
+// Created account on the Inquire via Phone modal (via email)
+analyticsHooks.on('mediator:auth:sign_up:success', ({context, user}) => {
```

Comment:
It is using a `mediator ` event. I actually wanted to use `analyticsHooks` for code pattern consistency, so I'll do that then.